static/app/components/group/pluginActions.tsx

```diff
@@ -39,6 +39,41 @@ type State = {
   pluginLoading: boolean;
 };
 
+export function openPluginActionModal({
```

Comment:
a lot of these are just making an exported function to open the modal that i can share